rllib/agents/impala/vtrace_torch_policy.py

```diff
@@ -170,7 +170,7 @@ def __init__(self):
         pass
 
     @override(TorchPolicyV2)
-    def optimizer(
+    def make_optimizer(
         self,
     ) -> Union[List["torch.optim.Optimizer"], "torch.optim.Optimizer"]:
         if self.config["opt_type"] == "adam":
```

Comment:
Here, it looks like that a custom optimizer is brought in via the configuration parameter `opt_type`